src/components/Modal/index.js

```diff
@@ -129,16 +129,35 @@ export default class Modal extends PureComponent {
     )
   }
 
+  isDescendant = (parent, child) => {
```

Comment:
would this `closest()` work instead of this? https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Reply:
yep, worked beautifully, thanks!